FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/portable/NetworkInterface/STM32Hxx/NetworkInterface.c

```diff
@@ -0,0 +1,557 @@
+/*
+ * Some constants, hardware definitions and comments taken from ST's HAL driver
+ * library, COPYRIGHT(c) 2015 STMicroelectronics.
+ */
+
+/*
+FreeRTOS+TCP V2.2.1
+Copyright (C) 2018 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
```

Comment:
2020

Reply:
I didn't know, can we always update the year in the Copyright (C) notice?

Reply:
Updated the year in a commit called "Update year of Copyright to 2020".